js/index.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function () { 
+
+	// // hide all answers/
+	$('#learnMoreText, #readMoreText, #read-less, #learn-less').addClass('hide');
+
+	$("#read-more").click(readMore);
+	$("#learn-more").click(learnMore);
+	$('#read-less').click(readLess);
+	$('#learn-less').click(learnLess);
+
+
+
+	function readMore () {
+		$("#readMoreText").slideDown();
+		$("read-more").slideDown();
```

Comment:
no need to do a `slideDown()` on the `#read-more`. Its already visible so all you wanna do at this point is `.hide()` it. Also, its missing the `#` in front of the ID so this actually wouldn't do anything anyway